### PR TITLE
feature flag pegin

### PR DESCRIPTION
**Pull request by ronaldsg20**
I've updated the session module on the store using the feature flags
		as getter, also including the pegin feature flag on the new design

### CODE REVIEW

File: src/pegin/components/create/PegInForm.vue [hunk 1]
```diff
@@ -153,6 +157,11 @@ export default defineComponent({
       selected.value = selectedType;
     }
 
+    onBeforeMount(() => {
+      const feature = flyoverFeature.value(FeatureNames.FLYOVER_PEG_IN);
+      flyoverEnabled.value = feature?.value === 'enabled';
+    });
+
```
Comment (lserra-iov) [inline src/pegin/components/create/PegInForm.vue:164]: Would make sense to add the same for peg-out [here](https://github.com/rsksmart/2wp-app/blob/feature/ff-pegin/src/pegout/views/PegOut.vue)?

**Review by lserra-iov — APPROVED** [review]

**Review by alexjavabraz — APPROVED** [review]
LGTM

File: src/pegin/components/create/PegInForm.vue (hunk 1 above)
Comment (ronaldsg20) [reply src/pegin/components/create/PegInForm.vue:164]: yes, I'll update it. Thanks

**Review by leoiovlabs — APPROVED** [review]
LGTM

**Pull request merged**